coretex/cli/project.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+import click
+from ..configuration import loadConfig, saveConfig
+from .. import Project, ProjectType
+
+
+def createProject(name: str) -> None:
+    try:
+        new_project = Project.createProject(name, ProjectType.other)
+        if new_project is not None:
+            config = loadConfig()
+            config["projectId"] = new_project.id
+            saveConfig(config)
```

Comment:
This is a duplicate functionality used both here, and in `select` command. Extract it into a separate function